packages/experiments-realm/productivity/task.gts

```diff
@@ -808,6 +809,7 @@ export class Task extends CardDef {
   @field dueDate = contains(DateField);
```

Comment:
We should remove the `dateStarted` and `dueDate` fields and use the new `dateRange` field instead. Also add an atom template to the dateRange component and move the UI that is used in the task cards with the calendar there.